doc/tutorial/climada_entity_Exposures_osm.ipynb

```diff
@@ -0,0 +1,692 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to get exposure data from OpenStreetMap\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "[OpenStreetMap data](https://www.openstreetmap.org/) is a freely-accessible and valuable data source which can provide information on the geolocations of a variety of assets such as critical infrastructures, buildings, or ecosystems. Such data can then be used within the risk modelling chain of CLIMADA as exposures. In this tutorial we show how to retrieve exposure data by querying the [OpenStreetMap database](https://www.openstreetmap.org/) using the [osm-flex](https://osm-flex.readthedocs.io/en/latest/?badge=latest) python package."
```

Comment:
The links to the other ressources in the documentation do not show up as links in the documentation.